modules/contact/src/main/kotlin/edu/stanford/spezi/modules/contact/ContactComposable.kt

```diff
@@ -76,14 +76,13 @@ fun ContactComposable(contact: Contact, modifier: Modifier = Modifier) {
                 horizontalArrangement = Arrangement.spacedBy(Spacings.small),
                 verticalAlignment = Alignment.CenterVertically
             ) {
-                val image = contact.image ?: Icons.Default.AccountBox
-                Icon(
-                    image,
+                val image = contact.image ?: ImageResource.Vector(Icons.Default.AccountBox)
+                ImageResourceIcon(
+                    imageResource = image,
                     contentDescription = stringResource(R.string.profile_picture),
-                    tint = Colors.primary,
                     modifier = Modifier
                         .size(Sizes.Icon.medium)
-                        .testIdentifier(ContactComposableTestIdentifier.IMAGE, image.name)
+                        .testIdentifier(ContactComposableTestIdentifier.IMAGE, image.identifier)
```

Comment:
with the imageResourceIdentifier proposed above there is no need to specify the test identifier as it sets the test tag by default and we can use the same id of the image to query the node